src/fides/api/ops/schemas/connection_configuration/connection_secrets_dynamodb.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from fides.api.ops.schemas.base_class import NoValidationSchema
+from fides.api.ops.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class DynamoDBSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed to connect to an Amazon DynamoDB cluster"""
+
+    region_name: str
+    aws_secret_access_key: str
+    aws_access_key_id: str
+
+    _required_components: List[str] = [
+        "region_name",
+        "aws_secret_access_key",
+        "aws_access_key_id",
+    ]
```

Comment:
A more succinct version of this would be something in the base class to autogenerate this property, so in cases like this where all components are required we don't need to redefine this. Probably not worth changing here though.

Reply:
Agreed, it felt like a better follow on issue here to look at both the way `ctl` and `ops` worked and unify them, will add a follow on issue 👍🏽

Reply:
Added in #3257